README.md

```diff
@@ -139,6 +139,12 @@ spec:
 
 ## Changelog
 
+### v0.2.1
+
+#### Fixed:
+
+* Non-existent fields (messageScope returns fields, even if the message do not contain these fields, instead of returning null)
```

Comment:
```suggestion
* Non-existent fields (`messageScope` returns a field, even if the message does not contain this field, instead of returning null)
```